### ISSUE TITLE
Quiz Time Limit

### ISSUE THREAD

**Issue by srinidhishankar** [dilevin/computer-graphics-csc317#70]
I just want to confirm that there is no time limit for the quiz? I wrote like half of it and stepped away bc it said that there's no time limit but just want to make sure thats ok. Thanks!

**Comment by dilevin** [comment]
Correct, no time limit Sent from my iPhoneOn Oct 1, 2024, at 5:47 PM, srinidhishankar ***@***.***> wrote:﻿
I just want to confirm that there is no time limit for the quiz? I wrote like half of it and stepped away bc it said that there's no time limit but just want to make sure thats ok. Thanks!
